_sass/_settings.scss

```diff
@@ -8,3 +8,6 @@ $color-navigation-active-bar: $ubuntu-orange;
 // Sidebar
 $content-padding: 0 5%;
 $sidebar-width: 18rem;
+
+// Other
+$breakpoint-medium:    875px
```

Comment:
You should use `$breakpoint-navigation-threshold` to change the menu breakpoint.